indicators/NO_FUNC_001-004/R/NO_FUNC_001-004.qmd

```diff
@@ -0,0 +1,3139 @@
+                                                  'T3-C4','T3-C7','T3-C10','T3-C13',
+                                                  'T3-C14','T7-C1','T7-C2','T7-C3',
+                                                  'T7-C6','T7-C8',
+                                                  'T7-C12','T7-C13C14','T7-C4','T7-C7C9','T7-C5','T7-C10',
+                                                  'T22-C1C3a','T22-C1C3b','T22-C2C4',
+                                                  
+                                                  'V3-C1a','V3-C1b','V3-C1c','V3-C1d','V3-C1e',
+                                                  'V1-C1a','V1-C1b','V1-C1c','V1-C1d','V1-C1e',
+                                                  'V1-C2a','V1-C2b','V1-C2c','V1-C2d',
+                                                  'V1-C3a','V1-C3b','V1-C3c','V1-C3d',
+                                                  'V1-C4a','V1-C4b','V1-C4c','V1-C4d',
+                                                  'V1-C4e','V1-C4f','V1-C4g','V1-C4h',
+                                                  'V3-C2','V1-C5',
+                                                  'V1-C6a','V1-C6b',
+                                                  'V1-C7a','V1-C7b',
+                                                  'V1-C8a','V1-C8b',
+                                                  'V2-C1a','V2-C1b',
+                                                  'V2-C2a','V2-C2b',
+                                                  'V2-C3a','V2-C3b',
+                                                  "V4-C2","V4-C3",
+                                                  'V8-C1','V8-C2','V8-C3',
+                                                  'T32-C1C2','T32-C3C4',
+                                                  
+                                                  'T32-C5C20a',
+                                                  'T32-C7C8',
+                                                  'T32-C5C20b',
+                                                  'T32-C9a',
+                                                  'T32-C9b',
+                                                  'T32-C15',
+                                                  'T32-C21C6a',
+                                                  'T32-C21C6b',
+                                                  'T32-C10a',
+                                                  'T32-C10b',
+                                                  'T32-C16',
+                                                  'T41a','T41b',
+                                                  'T45-C1C2','T45-C3',
+                                                  'V10-C1C2','V10-C3'
+)
+#head(NiN.wet_mount_forest_seminat)
+
+
+# translating the abundance classes into %-cover
+coverscale <- data.frame(orig=0:6,
+                         cov=c(0,1/32,1/8,3/8,0.6,4/5,1)
+)
+
+NiN.wet_mount_forest_seminat.cov <- NiN.wet_mount_forest_seminat
+colnames(NiN.wet_mount_forest_seminat.cov)
```

Comment:
this line should be commented out

Reply:
The chunk has result: hide so it's not a problem